docs/source/howto/level1-csd.rst

```diff
@@ -0,0 +1,312 @@
+.. _tut-l1csd:
+
+Level 1 Cloud Storage Downloader
+================================
+
+**Downloading Landsat and Sentinel-2 Level-1 data from cloud services**
+
+This tutorial covers the process of querying and downloading Landsat and Sentinel-2 Level 1 data from cloud services (currently Google Cloud Storage).
+FORCE Level 1 CSD offers a high level of detail in requesting data and creates a consistent Level 1 data pool, as well as queue files that can be used for Level 2 processing.
+
+.. admonition:: Info
+
+   *This tutorial uses FORCE v. 3.5*
+
+FORCE Level 1 CSD makes use of the Landsat and Sentinel-2 metadata catalogues from Google Cloud Storage to find scenes matching the user's requirements and downloads them through `gsutil <https://cloud.google.com/storage/docs/gsutil>`_. A check for scenes already existing locally is performed and only new scenes are downloaded.
+
+
+Getting started
+---------------
+
+gsutil configuration
+""""""""""""""""""""
+
+Currently, Landsat data can be downloaded without a login (this will change at some point), but downloading Sentinel-2 data requires authentication.
+This is done through gsutil and only needs to be done once before the first run. The credentials will be stored in a file called ``.boto`` in the user's home directory (make sure to check the notes at the end of this document if you're running FORCE in Docker).
+
+Type the following command to start the authorization process
+
+.. code-block:: none
+
+    gsutil config
+
+
+Follow the on-screen instructions to authorize access to the Google Cloud Platform APIs. A project name is not required, just hit enter or make up a name when asked.
+
+Downloading the metadata catalogues
+"""""""""""""""""""""""""""""""""""
+
+The metadata catalogues contain metadata for all scenes stored on Google Cloud Storage. This includes all the information we need to find data that matches our needs - location, date, cloud cover estimates, sensor, etc...
+The ``-u | --update`` option will download the latest (compressed) catalogues and extract them to the metadata directory specified by the user.
```

Comment:
I thought that the metadata catalogue is always global? 
I guess it is. The 2nd sentence in l. 39 confused me; probably delete this here? Make **all scenes** bold?
Probably add a sentence that this metadata catalogue is fairly huge and can be shared between users?